REFERENCE.md

```diff
@@ -45,6 +45,7 @@ The package method needs specific yum or apt repo settings which are not made ye
 * [`prometheus::server`](#prometheusserver): class to manage the actual prometheus server. This class gets called from the init.pp
 * [`prometheus::snmp_exporter`](#prometheussnmp_exporter): This module manages prometheus snmp_exporter
 * [`prometheus::statsd_exporter`](#prometheusstatsd_exporter): This module manages prometheus statsd_exporter
+* [`prometheus::unbound_exporter`](#prometheusunbound_exporter): This module manages prometheus node node_exporter
```

Comment:
Is this supposed to be in this PR?

Reply:
ah. that sneaked in while I was working on the other PR

Reply:
fixed it